tests/show_sflow_test.py

```diff
@@ -0,0 +1,71 @@
+import os
```

Comment:
can you rename show_sflow_test.py to sflow_test.py, because we need to add config sflow tests in the file later as well.

Reply:
sure, I am planning to add config soon too, Let me know if somebody is already on it.